ios_app_clip/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -296,6 +297,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		0A48A2782508C8E40036A529 /* Build Flutter */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
Uncheck `Based on dependency analysis` to avoid warning:
```
Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase.
```